packages/artifact_proxy/lib/config.dart

```diff
@@ -114,11 +114,4 @@ final flutterArtifactPatterns = {
   r'flutter_infra_release\/gradle-wrapper\/(.*)\/gradle-wrapper\.tgz',
   r'flutter_infra_release\/flutter\/fonts\/(.*)\/fonts\.zip',
   r'flutter_infra_release\/cipd\/flutter\/web\/canvaskit_bundle\/\+\/(.*)',
-  r'download.flutter.io\/org\/jacoco\/org.jacoco.ant\/0\.8\.3\/org.jacoco.ant-0\.8\.3\.pom',
```

Comment:
took a closer look at these and they aren't in the flutter storage bucket either so removing them from our config